### ISSUE TITLE
Sites getting classified as untargeted

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#9013]
## Description 

While testing #8646 found sites are getting classified as untargeted

## Steps to Reproduce

   1. Set device region to India
   2. Enable rewards and visit a bunch of sites
   3. All sites are categorised as untargeted

## Actual result:

```
[9972:2976:0403/153541.443:INFO:ads_service_impl.cc(2136)] AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T15:35:41Z","tabId":4,"tabType":"click","tabUrl":"https://brave.com/","tabClassification":["untargeted"]}}
[9972:2976:0403/154207.149:INFO:ads_service_impl.cc(2136)] AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T15:42:07Z","tabId":8,"tabType":"click","tabUrl":"https://www.imdb.com/","tabClassification":["untargeted"]}}
[9972:2976:0403/154207.149:INFO:ads_service_impl.cc(2136)] AdsService Event Log: AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T15:43:52Z","tabId":9,"tabType":"click","tabUrl":"https://www.law.com/?slreturn=20200303061348","tabClassification":["untargeted"]}}
```

## Expected result:
Sites should get classified correctly

## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 1.7.85 Chromium: 80.0.3987.162 (Official Build) dev (64-bit)
-- | --
Revision | f2c5dd6138153bb0658091205bd1a1717f16081a-refs/branch-heads/3987@{#1034}
OS | Windows 10 OS Version 1809 (Build 17763.1098)

## Version/Channel Information:

- Can you reproduce this issue with the current release? NA
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the dev channel? yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome?  NA

## Miscellaneous Information:

Reproduced by @GeetaSarvadnya 
cc: @tmancey @brave/legacy_qa

**Comment by GeetaSarvadnya** [comment]
Reproduced in windows 10 x64 - 1.7.82
```
[8788:13284:0403/185306.023:INFO:ads_service_impl.cc(2136)] AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T18:53:06Z","tabId":4,"tabType":"click","tabUrl":"https://brave.com/","tabClassification":["untargeted"]}}
7:01
[8788:13284:0403/185352.987:INFO:ads_service_impl.cc(2136)] AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T18:53:52Z","tabId":5,"tabType":"click","tabUrl":"https://www.imdb.com/","tabClassification":["untargeted"]}} (edited) 
7:01
[8788:13284:0403/185609.303:INFO:ads_service_impl.cc(2136)] AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T18:56:09Z","tabId":7,"tabType":"click","tabUrl":"https://www.mymoneyblog.com/","tabClassification":["untargeted"]}} 
````

**Comment by LaurenWags** [comment]
Doesn't reproduce for me on macOS (my region is US though and we have a language model for English)

```
[7929:775:0403/130732.893019:INFO:ads_service_impl.cc(2136)] AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T13:07:32Z","tabId":20,"tabType":"click","tabUrl":"https://brave.com/","tabClassification":["technology & computing","technology & computing"]
```

```
[7929:775:0403/130933.256162:INFO:ads_service_impl.cc(2136)] AdsService Event Log: {"data":{"type":"load","timestamp":"2020-04-03T13:09:33Z","tabId":12,"tabType":"click","tabUrl":"https://www.instyle.com/fashion","tabClassification":["fashion","fashion"]
```


Brave | 1.7.85 Chromium: 80.0.3987.162 (Official Build) dev (64-bit)
-- | --
Revision | f2c5dd6138153bb0658091205bd1a1717f16081a-refs/branch-heads/3987@{#1034}
OS | macOS Version 10.14.6 (Build 18G3020)

**Comment by tmancey** [comment]
@LaurenWags @srirambv @GeetaSarvadnya This was by-design as India does not use targeted advertising at this time, however after speaking with @jsecretan we have agreed if the locale is en_IN we will use the English model at this time for page classification. This has been resolved as part of https://github.com/brave/brave-core/pull/5158/files

**Comment by srirambv** [comment]
Closing as dupe of #7815 which is fixed by #5158 as per https://github.com/brave/brave-browser/issues/9013#issuecomment-609008727
